metaflow/metaflow_config.py

```diff
@@ -398,9 +398,13 @@
 
 OTEL_ENDPOINT = from_conf("OTEL_ENDPOINT")
 ZIPKIN_ENDPOINT = from_conf("ZIPKIN_ENDPOINT")
+# URL template for tracing links in the UI.
+# An example value is "https://zipkin.example.com/trace/{trace_id}".
```

Comment:
nit: I would add that it needs to contain {trace_id} where the trace ID should go.

Reply:
@savingoyal How does the tracing URL look like in OB's Otel setup? Trying to see if it makes more sense to keep it in OSS config or Netflix internal config